main.go

```diff
@@ -239,18 +246,26 @@ func Run(tokenFlag string, printHelp func(bool) func()) error {
 		return fmt.Errorf("%s", err)
 	}
 	if len(workspaces) == 0 {
-		return fmt.Errorf("no workspace found")
+		return fmt.Errorf("no workspaces found")
+	}
+
+	workspaceName := workspaceFlag
+	if workspaceName == "" {
+		workspaceName = conf.Workspace
 	}
```

Comment:
suggestion: I like that in most CLIs, passing a flag "overrides" any env var, and any env var overrides the configuration file. What do you think?

In this context, that would mean inverting the order:

```suggestion
	workspaceName := conf.Workspace
	if workspaceName == "" {
		workspaceName = workspaceFlag
	}
```